_data/sites.json

```
@@ -7714,6 +7714,17 @@
         ]
     },
 
+    {
+        "name": "pdfFiller",
+        "url": "https://www.pdffiller.com/en/account/settings",
+        "difficulty": "easy",
+        "notes": "Log in to your account, then go to 'Settings' tab and click 'Delete Account'.",
+        "notes_pl": "Zaloguj się na swoje konto, potem przejdź do 'Settings' i kliknij 'Delete Account'.",
+        "domains": [
+            "www.pdffiller.com"
```

```suggestion
            "pdffiller.com"
```
We don't need the www for domains